whois-commons/src/main/resources/applicationContext-commons.xml

```diff
@@ -17,11 +17,11 @@
     <!-- @Transactional CAN be used on private methods with mode="aspectj". However, annotations on interfaces will NOT work! -->
     <tx:annotation-driven mode="aspectj"/>
 
-    <context:spring-configured/>
+    <context:spring-configured />
     <context:component-scan base-package="net.ripe.db.whois.common"/>
 
     <context:annotation-config/>
-    <cache:annotation-driven/>
+    <cache:annotation-driven mode="aspectj"/>
```

Comment:
Can we use the annotation @EnableCaching instead?

Reply:
Yes, according to http://www.springframework.org/schema/cache/spring-cache.xsd :
```
	Indicates that cache configuration is defined by annotations on bean classes,
	and that proxies are automatically to be created for the relevant annotated beans.

	The default annotations supported are Spring's @Cacheable, @CachePut and @CacheEvict.
	If spring-context-support and the JSR-107 API are on the classpath, additional proxies
	are automatically created for JSR-107 annotated beans, that is @CacheResult, @CachePut,
	@CacheRemove and @CacheRemoveAll.

	See org.springframework.cache.annotation.EnableCaching javadoc for information on
	code-based alternatives to this XML element.
```
But we should keep the "aspectj" mode declaration

Reply:
Yes , @EnableCaching(mode = AdviceMode.ASPECTJ)?

Reply:
Yes sounds good

Reply:
Done